### PR TITLE
Monster App Template

### PR DESCRIPTION
**Pull request by agubler**

### CODE REVIEW

File: monster-cards/tests/intern.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+// Default desired capabilities for all environments. Individual capabilities can be overridden by any of the
+// specified browser environments in the `environments` array below as well. See
+// https://code.google.com/p/selenium/wiki/DesiredCapabilities for standard Selenium capabilities and
+// https://saucelabs.com/docs/additional-config#desired-capabilities for Sauce Labs capabilities.
+// Note that the `build` capability will be filled in with the current commit ID from the Travis CI environment
+// automatically
+export const capabilities = {
+	'browserstack.debug': false,
+	project: 'Dojo 2',
+	name: 'dojo-<< package-name >>'
```
Comment (matt-gadd) [inline monster-cards/tests/intern.ts]: package name here

File: monster-cards/tests/support/util.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (matt-gadd) [inline monster-cards/tests/support/util.ts]: do we actually ever use this?

File: monster-cards/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	"compilerOptions": {
+		"declaration": false,
+		"module": "umd",
+		"noImplicitAny": true,
+		"outDir": "_build/",
+		"baseUrl": "node_modules",
+		"removeComments": false,
+		"sourceMap": true,
+		"target": "es5",
+		"moduleResolution": "classic"
```
Comment (matt-gadd) [inline monster-cards/tsconfig.json]: lets change this to `node`, it shouldn't in theory affect anything at this point

File: monster-cards/typings.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+	"name": "dojo2-monster-cards",
+	"dependencies": {
+		"@reactivex/RxJS": "npm:@reactivex/rxjs",
+		"immutable": "npm:immutable",
```
Comment (matt-gadd) [inline monster-cards/typings.json]: wth `node` resolution, are these required still?

File: monster-cards/tests/support/util.ts (hunk 2 above)
Comment (agubler) [inline monster-cards/tests/support/util.ts]: I don't know, it is in the package template...

File: monster-cards/tests/unit/main.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import * as main from 'src/main';
```
Comment (matt-gadd) [inline monster-cards/tests/unit/main.ts]: if you switch to `node` resolution you will need to change this to a relative path

File: monster-cards/src/index.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+	<head>
+		<meta charset="utf-8">
+		<title>dojo2 Monster Cards</title>
+	</head>
+	<body>
+		<script src="./../../node_modules/dojo-loader/dist/umd/loader.min.js"></script>
+		<script src="./index.js"></script>
+			<header is="app-projector">
+				<div>
+					<dojo-widget data-options='{"id":"header-1"}'>
```
Comment (rishson) [inline monster-cards/src/index.html]: Ids are mandatory? If so, how about a convention of [page]-[widget], e.g. home-header, shoppingCart-footer?

Comment (agubler) [inline monster-cards/src/index.html]: This is just a place-holder, nothing to apply convention quite yet.

File: monster-cards/tests/unit/main.ts [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
```
Comment (agubler) [inline monster-cards/tests/unit/main.ts]: Fails to resolve this with `node` resolution

Comment (agubler) [inline monster-cards/tests/unit/main.ts]: ignore me, still have to install typings :trollface:

**Pull request merged**